interfaces/Output_from_core.atd

```diff
@@ -207,7 +225,8 @@ type target_time = {
   run_time: float;
 }
 
-type rule_times = {
+type rule_times
+  <python decorator="dataclass(frozen=True)"> = {
```

Comment:
maybe this dataclass(frozen=True) should be the default in atdpy? so we don't need to specify it each time?